profiles/common/modules/features/nexteuropa_dgt_connector/tmgmt_poetry/tmgmt_poetry.module

```diff
@@ -711,70 +709,418 @@ function tmgmt_poetry_form_tmgmt_entity_ui_translate_form_alter(&$form, &$form_s
-            return;
-          }
-
-          // Check the poetry status.
-          $result = db_select('poetry_status', 'ps')
-            ->fields('ps')
-            ->condition('tjiid', $existing_job->tjiid, '=')
-            ->execute()
-            ->fetchAssoc();
-
-          if (isset($result['status']) && $result['status'] == POETRY_STATUS_MESSAGE_PEN) {
-            $translation_possible = FALSE;
+        // If translatable languages are available add the button and other
+        // elements for the additional languages requests functionality.
+        if (_tmgmt_poetry_check_translatability($form_state['tmgmt_cart']['item_id'])) {
+          // Checking and getting additional translatable languages.
+          if (!empty($trans_langs = _tmgmt_poetry_get_translatable_languages($existing_jobs, $form, $form_state))) {
+            // Injecting 'request additional languages' functionality form
+            // elements.
+            _tmgmt_poetry_inject_add_new_languages_elements($form, $trans_langs);
           }
         }
-
-        // Force form to allow user actions.
-        $form_state['force_actions'] = TRUE;
-
-        if ($translation_possible) {
-          // Add the new submit button.
+      }
+      else {
+        // Provide 'Request translation' action or hide actions.
+        // NEXTEUROPA-7719 - Simplifying Poetry UI - as a part of this ticket
+        // there is new #markup element which will be displayed every time when
+        // there is a translation request on going.
+        if ($translation_possible && !$existing_jobs) {
+          $form['actions']['#type'] = 'actions';
           $form['actions']['request'] = [
             '#type' => 'submit',
-            '#value' => t('Request translation update'),
-            '#submit' => ['tmgmt_workbench_translate_form_submit'],
-            '#validate' => ['tmgmt_workbench_translate_form_validate'],
-            '#limit_validation_errors' => array(
-              array('languages'),
-            ),
+            '#value' => t('Request translation'),
+            '#submit' => ['_tmgmt_poetry_workbench_translate_form_submit'],
+            '#validate' => ['_tmgmt_poetry_workbench_translate_form_validate'],
           ];
         }
         else {
-          // Disable all actions.
           unset($form['actions']);
-          $lang_codes = array_keys($form['languages']['#options']);
           foreach ($lang_codes as $lang_code) {
             $form['languages'][$lang_code] = [
               '#type' => 'checkbox',
               '#disabled' => TRUE,
             ];
           }
-          $form['languages_description'] = [
-            '#type' => 'item',
-            '#prefix' => '<div class="messages warning">',
-            '#markup' => t('Please wait the acceptation translation process before update request.'),
-            '#suffix' => '</div>',
-          ];
+          if ($existing_jobs && !isset($form['languages_description'])) {
```

Comment:
$existing_jobs is empty here, so what is inside will never run (see comment above).